docs/analytics_welcome/03_how_we_work.md

```diff
@@ -1,21 +1,74 @@
 # How We Work (WIP)
+
 ## Meetings
-### Week 1
+
+### Existing Meetings
```

Comment:
Similar to below, can we turn meetings into a table? (feel free to change column names, anything about table for clarity, etc.. )

| meeting | when | purpose | 
| -------- | ------ | -------- |
| Technical onboarding | Schedule with X | Ensure you have access to tools. Go over best practices |
| Data Services All-hands | Weekly | |
| Lunch and Learn | Weekly | |
| Analyst Stand-ups | 15 minutes Daily | |

(Technical onboarding could be moved out etc.. if useful, threw table down quickly as an example..!)

Reply:
collapsed nested headers in meetings section to tables like the example above